elide-model-config/src/main/java/com/yahoo/elide/modelconfig/validator/DynamicConfigValidator.java

```diff
@@ -194,6 +194,9 @@ public void readConfigs() throws IOException {
         readConfigs(fileLoader.loadResources());
     }
 
+    /**
+     * @throws IOException
```

Comment:
@aklish is there a reason this function throws IOException when currently it does not?

Reply:
No